webapp/test/App.spec.js

```diff
@@ -1,24 +1,56 @@
 import { mount } from '@vue/test-utils';
 import app from '../src/App.vue';
+import {USER, AUTH_TOKEN} from '../src/constants/settings.js'
 
-describe('App.vue', () => {
+const title = "Frontend-Einkaufslisten-Bearbeitungs-Und-Erstellungsmaschine";
+const notLoggedInInfo = "You are not logged in.";
+const loginFormDivName = "loginForm";
+const todoListDivName = "todoList";
 
+describe('App.vue', () => {
   let wrapper = mount(app);
-  beforeEach(() => {
-    wrapper = mount(app);
-  });
 
-  it('renders the todo app', () =>{
-    expect(wrapper.contains('div')).toBe(true);
-  });
+  describe("When App is loaded", () => {
+    beforeEach(() => {
+      wrapper = mount(app);
+    });
+
+    it("renders the login form", () => {
+      const loginForm = wrapper.find({name: loginFormDivName});
```

Comment:
I don't think you save anything with the variable name. You could also just write down "loginForm" and save me one lookup while reading.